### DISCUSSION TITLE
nautilus-ibapi port error

### DISCUSSION THREAD

**Discussion by paulhwatson** [nautechsystems/nautilus_trader#1778] [Q&A] [answered]
Hi guys. I'm trying to get an Interactive Brokers streaming data service running using Docker. I have an up to date image loaded into docker desktop on my apple silicon Mac, but when I play it fails with a (HTTP code 500) server error - Ports are not available: exposing port TCP 127.0.0.1:5900 -> 0.0.0.0:0: listen tcp 127.0.0.1:5900: bind: address already in use.
Does ibapi need to open this port? 
Many thanks Paul

**Comment by cjdsellers** [comment]
Hi @paulhwatson

Thanks for the report.

Was this just a simple local setup issue in the end? or any pointers on this @rsmb7z :pray:

**Comment by paulhwatson** [comment, marked as answer]
Setup issue. I’ve downloaded a different build as per advice and works fine now 😊On 1 Sep 2024, at 2:27 am, Chris Sellers ***@***.***> wrote:﻿
Hi @paulhwatson
Thanks for the report.
Was this just a simple local setup issue in the end? or any pointers on this @rsmb7z 🙏
